server/__init__.py

```diff
@@ -98,3 +131,8 @@ class Root(object):
     info['apiRoot'].image = api.ImageResource()
     info['apiRoot'].segmentation = api.SegmentationResource()
     info['apiRoot'].study = api.StudyResource()
+
+    # Move girder app to /girder, serve isic_archive app from /
+    info['serverRoot'], info['serverRoot'].girder = (Webroot(),
```

Comment:
Let's keep the Girder app mounted at `/`. Mount this new app at `/isic` for now.